wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/server/AuthorizationSupplier.java

```diff
@@ -14,8 +14,11 @@
 import org.eclipse.che.api.factory.server.scm.exception.ScmCommunicationException;
 import org.eclipse.che.api.factory.server.scm.exception.ScmUnauthorizedException;
 
-/** Compute the Authorization header to sign the OAuth 1 request. */
-public interface AuthorizationHeaderSupplier {
+public interface AuthorizationSupplier {
```

Comment:
Looks like only responsibility of this interface is to calculate the headers. TBH it's not clear to me what does it mean to "compute" the headers. Looking at `AuthorizationSupplier` and `OAuthAuthenticator` I'm a bit confused what are exact responsibilities of these classes and their relations. Probably not the scope of this issue though.